StackOverflow/Model/DataDelegate.h

```diff
@@ -0,0 +1,16 @@
+//
+//  DataDelegate.h
+//  StackOverflow
+//
+//  Created by Malea on 2017/10/04.
+//  Copyright © 2017 DVT. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@protocol DataDelegate <NSObject>
+
+- (void)receivedDataJSON:(NSData *)objectNotation;
```

Comment:
These should be defined as required since the delegate can't function as a delegate without them.